### PR TITLE
fixed login

### PR DESCRIPTION
**Pull request by Geofery**
Changed login response to include user--> parent --> family and address --> location.

### CODE REVIEW

File: src/Controllers/userController.ts [hunk 1]
```diff
@@ -61,13 +61,17 @@ export const getAllUsers = async (req: Request, res: Response) => {
 
 //TODO Denne kan ikke tage imod rolle uden at opdatere email, crasher appen.
 export const updateUser = async (req: Request, res: Response) => {
-	const userRequestDTO: UserRequestDTO = {
+	console.log(req.body);
```
Comment (ThonyDK) [inline src/Controllers/userController.ts:64]: Tænker måske console.log skal væk

File: src/Services/userService.ts [hunk 2]
```diff
@@ -79,23 +83,22 @@ export const getParentByEmailAndPassword = async (
...
-      parent: {
-        firstName: response.parent.firstName,
-        lastName: response.parent.lastName,
-        age: response.parent.age,
-        gender: response.parent.gender,
-        parentId: response.parent.parentId,
-        address: response.parent.address,
-      },
-      userActive: true,
+      //God kodestil...
```
Comment (Rasmus-Kibshede) [inline src/Services/userService.ts:91]: remove comments

File: src/Services/userService.ts [hunk 3]
```diff
@@ -79,23 +83,22 @@ export const getParentByEmailAndPassword = async (
...
+      parent: await parentRepo.findOneByID(response.parent.parentId) as unknown as ParentResponseDTO,
+      userActive: false
     };
 
+    //God kodestil...
```
Comment (Rasmus-Kibshede) [inline src/Services/userService.ts:96]: remove comments

File: src/Services/userService.ts [hunk 4]
```diff
@@ -79,23 +83,22 @@ export const getParentByEmailAndPassword = async (
...
-        firstName: response.parent.firstName,
-        lastName: response.parent.lastName,
-        age: response.parent.age,
-        gender: response.parent.gender,
-        parentId: response.parent.parentId,
-        address: response.parent.address,
-      },
-      userActive: true,
+      //God kodestil...
+      parent: await parentRepo.findOneByID(response.parent.parentId) as unknown as ParentResponseDTO,
```
Comment (Rasmus-Kibshede) [inline src/Services/userService.ts:92]:
```suggestion
      parent: await parentRepo.findOneByID(response.parent.parentId) as ParentResponseDTO,
```

File: src/Services/userService.ts [hunk 5]
```diff
@@ -79,23 +83,22 @@ export const getParentByEmailAndPassword = async (
...
+      userActive: false
     };
 
+    //God kodestil...
+  user.parent!.address = await addressRepo.findOneByID(Number(user.parent?.address.addressId)) as unknown as AddressResponseDTO;
```
Comment (Rasmus-Kibshede) [inline src/Services/userService.ts:97]:
```suggestion
  user.parent!.address = await addressRepo.findOneByID(Number(user.parent?.address.addressId)) as AddressResponseDTO;  
```

**Review by Rasmus-Kibshede — APPROVED** [review]

**Pull request merged**